src/server/worker/api_server.go

```diff
@@ -464,30 +463,26 @@ func (a *APIServer) downloadData(pachClient *client.APIClient, logger *taggedLog
 	if err := os.MkdirAll(filepath.Join(dir, "out"), 0777); err != nil {
 		return "", err
 	}
-	var incremental bool
-	if parentTag != nil {
-		if err := func() error {
-			var buffer bytes.Buffer
-			if err := pachClient.GetTag(parentTag.Name, &buffer); err != nil {
-				// This likely means that the parent job errored in some way,
-				// this doesn't prevent us from running the job, it just means
-				// we have to run it in an un-incremental fashion, as if this
-				// were the first input commit.
-				return nil
-			}
-			incremental = true
-			tree, err := hashtree.Deserialize(buffer.Bytes())
-			if err != nil {
-				return fmt.Errorf("failed to deserialize parent hashtree: %v", err)
-			}
-			if err := puller.PullTree(pachClient, path.Join(dir, "out"), tree, false, concurrency); err != nil {
-				return fmt.Errorf("error pulling output tree: %+v", err)
-			}
-			return nil
-		}(); err != nil {
-			return "", err
-		}
-	}
+	//var incremental bool
+	//if parentTag != nil {
+	//	if err := func() error {
+	//		tree, err := hashtree.GetHashTreeTag(pachClient, a.hashtreeStorage, parentTag)
+	//		if err != nil {
+	//			// This likely means that the parent job errored in some way,
+	//			// this doesn't prevent us from running the job, it just means
+	//			// we have to run it in an un-incremental fashion, as if this
+	//			// were the first input commit.
+	//			return nil
+	//		}
+	//		incremental = true
+	//		if err := puller.PullTree(pachClient, path.Join(dir, "out"), tree, false, concurrency); err != nil {
+	//			return fmt.Errorf("error pulling output tree: %+v", err)
+	//		}
+	//		return nil
+	//	}(); err != nil {
+	//		return "", err
+	//	}
+	//}
```

Comment:
This will be removed as part of `incremental` removal as well I assume.

Reply:
It did.